hack/e2e-common.sh

```diff
@@ -161,7 +161,7 @@ function save_hive_logs() {
 }
 # The consumer of this lib can set up its own exit trap, but this basic one will at least help
 # debug e.g. problems from `make deploy` and managed DNS setup.
-trap 'kill %1; save_hive_logs' EXIT
+trap 'set +e; kill %1; save_hive_logs' EXIT
```

Comment:
Shouldn't it be `set -e`? I've never seen it with a + before

Reply:
`set -e` turns error-on-exit *on*; `set +e` turns it *off*.

https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html <== see 3rd paragraph from the bottom.